templates/include/glean/custom_distribution.1.schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "type": "object",
+  "properties": {
+    "range": {
+      "type": "array",
+      "minItems": 2,
+      "maxItems": 2,
+      "items": {
+        "type": "number"
+      }
+    },
+    "bucket_count": {
+      "type": "integer"
+    },
+    "histogram_type": {
```

Comment:
Interesting, `histogram_type` is an int on desktop. Is this translation to a string happening in GV or Glean?

Reply:
Ok, so here, we've gone against desktop. :(